src/client/src/modals/ViewPlatformRequirementsModal/RequirementItem/messages.ts

```diff
@@ -0,0 +1,13 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  requirementInstalled: {
+    id: "ViewPlatformRequirementsModal.requirementItem.requirementInstalled",
```

Comment:
again, only if we decide renaming, remove the "View"

Reply:
Done.